README.md

```diff
@@ -71,14 +73,13 @@
 
 1. Test the connection to Kubernetes by executing the following:
     ```
-    export KUBECONFIG="$(pwd)/bootkube-assets/auth/kubeconfig"
-    kubectl get all --all-namespaces
+    cp $(pwd)/bootkube-assets/auth/user-config ./kubeconfig
```

Comment:
Could/should we add instructions here to commit the generated kubeconfig (which IIUC is not sensitive for this IAM flow) into the cluster's directory so that we can easily fetch/reference/use it later?